packages/coil-puppeteer-utils/src/lib/logoutCoil.ts

```diff
@@ -9,11 +10,18 @@ export async function logoutCoil(coilPage: Page): Promise<Page> {
   await coilPage.goto(`${COIL_DOMAIN}/settings`)
   await timeout(2e3)
 
-  const menuSelector = `svg[data-cy='hamburger-toggle']`
-  const logoutSelector = "//p[contains(., 'Log out')]"
+  await addCloudFlareAccessHeaders(coilPage)
+
+  const menuSelector = `[data-testid='MenuRoundedIcon']`
+  const signButton = "[data-testid='signout-nav-button']"
 
   try {
     await coilPage.click(menuSelector, { delay: 5e3, clickCount: 1 })
+    const [logoutBtn] = await coilPage.$x(
+      `//div[contains(@class, "MuiListItemButton-root") and contains(., "Sign out")]`
+    )
+    await logoutBtn.click()
+    return coilPage
```

Comment:
The catch is now catching multiple possible errors, but simply logging "failed to open hamburger menu"